src/graphql/mirror.js

```diff
@@ -318,6 +318,18 @@ export class Mirror {
           CREATE INDEX idx_connection_entries__connection_id
           ON connection_entries (connection_id)
         `,
+        dedent`\
+          CREATE TABLE network_log (
+              rowid INTEGER PRIMARY KEY,
+              query TEXT NOT NULL,
+              query_parameters TEXT NOT NULL,
+              request_timestamp INTEGER NOT NULL,
```

Comment:
Consider calling this `request_time_epoch_millis` (and likewise for the
response), as in the `updates` table, to make the units evident?

Reply:
updated